docs/docs/helm.mdx

```diff
@@ -0,0 +1,186 @@
+> can find your vendored Charts.
+
+<br />
+
+Once called, the `$.grafana` key holds the individual resources of Helm Chart as
+a regular Jsonnet object that looks roughly like so:
+
+```jsonnet
+{
+  cluster_role_binding_grafana_clusterrolebinding: {/* ... */},
+  cluster_role_grafana_clusterrole: {/* ... */},
+  config_map_grafana: {/* ... */},
+  config_map_grafana_test: {/* ... */},
+  deployment_grafana: {/* ... */},
+  // ...
+}
+```
+
+## Vendoring Helm Charts
+
+A core concept of Grafana Tanka is purity, which it **always yields the same
+resources**, regardless of surroundings. This is the case, when the project is
+strictly self-contained.
+
+Helm however keeps Charts and repository configuration somewhere around
+`~/.config/helm`, which violates above requirement.
+
+As such, we decided to only allow Helm Charts that can be found **inside the
+bounds of a project**. This means, you MUST put your Charts **somewhere next to
+the file that calls `helm.template()`**, so that it can be referred to using a
+relative path.
+
+### Vendor Location
+
+Where to actually put them inside the project is up to you, but keep in mind you
+need to refer to them using relative paths.
+
+We recommend always writing libraries that wrap the actual Helm Chart, so the
+consumer does not need to be aware of it. Whether you put these into `lib/` or
+publish them on GitHub (`vendor/`) is up to you.
+
+A library usually looks like this:
+
+```
+  /jsonnetfile.json
+  /main.libsonnet
+```
+
+When adopting Helm inside it, we recommend vendoring at the top level, as such:
+
+```diff
+  /jsonnetfile.json
+  /main.libsonnet
++ /charts
++ /charts/<someChart>
+```
+
+This way, you can refer to the charts as `./charts/<someChart>` from inside
+`main.libsonnet`. By keeping the Chart as close to the consumer as possible, the
+library is kept portable.
+
+### Charttool
+
+Helm does not make vendoring incredibly easy. `helm pull` provides the required
+plumbing, but it does not record its actions in a reproducible manner.
+
+Tanka ships a special utility at `tk tool charts`, which automates `helm pull`:
+
+```bash
+# Create a chartfile.yaml in the current directory, e.g. in lib/myLibrary
+$ tk tool charts init
+
+# Install the MySQL Chart at version 1.6.7 from the stable repository
+$ tk tool charts add stable/mysql@1.6.7
+```
+
+**Adding Charts:** To add a Chart, use the following:
+
+```bash
+$ tk tool charts add <repo>/<name>@<version>
+```
+
+This will also call `tk tool charts vendor`, so that the `charts/` directory is updated.
+
+<br />
+
+**Adding Repositories:**
+By default, the `stable` repository is automatically set up for you. If you wish
+to add another repository, you currently need modify `chartfile.yaml` directly
+to add it:
+
+```diff
+version: 1
+repositories:
+  - name: stable
+    url: https://kubernetes-charts.storage.googleapis.com
++ - name: grafana
++   url: https://https://grafana.github.io/helm-charts
+```
+
+## Troubleshooting
+
```

Comment:
Can you add a topic that covers what happens if you don't have `helm` installed? People might be searching for the error message.